impersonate_login/models/impersonate_log.py

```diff
@@ -11,7 +11,7 @@ class ImpersonateLog(models.Model):
     _description = "Impersonate Logs"
 
     user_id = fields.Many2one(
-        comodel_name="res.partner",
+        comodel_name="res.users",
```

Comment:
does it require any migration for existing records?

Reply:
@simahawk this is the fix already merged in V16 https://github.com/OCA/server-auth/commit/51eccf805efebcdd1f3c0141619a76a88991bcea

The integer written in `user_id` is a `res.users` anyway, so no need of migration